src/umockdev-pcap.vala

```diff
@@ -256,8 +272,11 @@ internal class IoctlUsbPcapHandler : IoctlBase {
                         urb_buffer_length -= 8;
                     }
 
+                    /* libusb always sets URB_CONTROL endpoint to 0x00, but the
+                     * kernel exposes it as 0x80/0x00 depending on the direction
```

Comment:
FWIW, the direction is a flag, the endpoint is only 4 bits:
https://www.beyondlogic.org/usbnutshell/usb6.shtml

Reply:
Yeah. The funny thing is that `bmRequestType` and the endpoint byte of other packets are very similar yet different. So it would be correct to just copy the top bit from the setup data into the endpoint, but we can also just ignore it.